geoportal/tests/functional/test_themes_mixed.py

```diff
@@ -172,15 +200,23 @@ def _only_name(self, item, attributes=None):
     @staticmethod
     def _get_filtered_errors(themes):
         errors = themes["errors"]
+
         errors = [
             e
             for e in errors
             if e != "The layer '' (__test_layer_external_wms) is not defined in WMS capabilities"
         ]
+
+        regex = re.compile(
+            r"The (GeoMapFish|WMS) layer name '[a-z0-9_]*', cannot be two times in the same block \(first level group\)."
+        )
+        errors = [e for e in errors if not regex.match(e)]
+
         regex = re.compile(
-            r"The (GeoMapFish|WMS) layer name '[a-z0-9_]*', cannot be two times in the same block (first level group)."
+            r"^WARNING! an error \'The WMS version \(\d\.\d\.\d\) you requested is not implemented."
         )
-        errors = [e for e in errors if regex.match(e)]
+        errors = [e for e in errors if not regex.match(e)]
```

Comment:
For me, this method shouldn't be changed and after we should test somthing like:
```
self.assertEqual(self._get_filtered_errors(themes), {"WARNING! an error 'The WMS version (1.0.0) you requested is not implemented."})
```